content/influxdb/clustered/reference/cli/influxctl/query.md

```diff
@@ -323,19 +325,74 @@ cat ./query.sql | influxctl query \
 ### Query InfluxDB v3 using credentials from the connection profile
 
 The following example uses the `database` and `token` defined in the `default`
-[connection profile](/influxdb/clustered/reference/cli/influxctl/#configure-connection-profiles).
+[connection profile](/influxdb/cloud-dedicated/reference/cli/influxctl/#configure-connection-profiles).
```

Comment:
Is this intentional?

Reply:
No, it's copy pasta. I just need to slow down  🤦🏻. Thanks for finding this.